http/headers/feature-policy.json

```diff
@@ -564,12 +564,12 @@
             }
           }
         },
-        "gyroscope": {
+        "layout-animations": {
           "__compat": {
-            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/gyroscope",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/layout-animations",
             "support": {
               "chrome": {
-                "version_added": "69",
+                "version_added": "72",
```

Comment:
Please do not add features for unreleased versions of Chrome, even if they are in Canary. Please remove throughout.

Reply:
Done, thanks.

Reply:
Thank you!